operator/cmd/mesh/testdata/manifest-generate/input/ingressgateway_k8s_settings.yaml

```diff
@@ -23,5 +23,7 @@ spec:
           - name: foo
             port: 1234
             targetPort: 1234
+        serviceAnnotations:
+          manifest-generate: "testserviceAnnotation"
```

Comment:
we don't seem to be testing k8s overlays for addonComponents, so hard to tell if this change is working.

Reply:
done, added test